src/ert/_c_wrappers/enkf/res_config.py

```diff
@@ -418,6 +422,13 @@ def _alloc_from_dict(self, config_dict):
             job = self.site_config.job_list.get_job_copy(
                 job_description[ConfigKeys.NAME]
             )
+            if job is None:
+                err_msg = (
+                    f"Could not find job `{job_description[ConfigKeys.NAME]}` "
+                    "in list of installed jobs: "
+                )
+                err_msg += f"{list(self.site_config.job_list)}"
+                raise ValueError(err_msg)
```

Comment:
these two new value errors - they will change the current behavior correct? I guess in a good way, but just wanted to raise a couple of questions. How will this be presented to our users? Should we handle the exception further up in the stack raised here?

Reply:
If this replace a `util_abort` this is great stuff and an improvement! :tada:

Reply:
thank you : )